.github/workflows/changelog.yml

```
@@ -0,0 +1,94 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+
+permissions:
+  contents: read
+
+env:
+  # Also change CACHE_VERSION in the other workflows
+  CACHE_VERSION: 8
+  DEFAULT_PYTHON: 3.8
```

```suggestion
  DEFAULT_PYTHON: 3.10
```

I was thinking of doing this for other workflows as well, but since `3.10` is faster than `3.8` I think it makes sense to run jobs on the latest stable versions.